src/main/kotlin/no/nav/dagpenger/oppslag/arena/ArenaClientSoap.kt

```diff
@@ -1,24 +1,27 @@
 package no.nav.dagpenger.oppslag.arena
 
+import no.nav.arena.services.lib.sakvedtak.SaksInfo
+import no.nav.arena.services.lib.sakvedtak.SaksInfoListe
+import no.nav.arena.services.sakvedtakservice.Bruker
+import no.nav.arena.services.sakvedtakservice.SakVedtakPortType
 import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.binding.BehandleArbeidOgAktivitetOppgaveV1
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Oppgave
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Oppgavetype
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Person
-import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.Tema
+import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.informasjon.*
 import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.meldinger.BestillOppgaveRequest
 import no.nav.tjeneste.virksomhet.behandlearbeidogaktivitetoppgave.v1.meldinger.BestillOppgaveResponse
 import java.util.Calendar
 import java.util.GregorianCalendar
 import javax.xml.datatype.DatatypeFactory
+import javax.xml.ws.Holder
 
-class ArenaClientSoap(val oppgaveV1: BehandleArbeidOgAktivitetOppgaveV1) {
+class ArenaClientSoap(val oppgaveV1: BehandleArbeidOgAktivitetOppgaveV1, val hentsak: SakVedtakPortType) {
     fun bestillOppgave(behandlendeEnhetId: String, fødselsnummer: String): String {
         val request = BestillOppgaveRequest()
 
         request.oppgavetype = Oppgavetype().apply { value = "STARTVEDTAK" }
 
         request.oppgave = Oppgave().apply {
             tema = Tema().apply { value = "DAG" }
```

Comment:
I'm not very sure how this plays with these SOAP-objects, but is not possible to simply do:
```suggestion
            tema = Tema().value = "DAG"
```
or
```suggestion
            tema = Tema().setValue("DAG")
```
or even
```suggestion
            tema = Tema("DAG")
```

It became very verbose with apply on every value here.

Reply:
Sadly, none of the approaches suggested seem to work with the way these SOAP-objects are generated :(

`tema = Tema().value = "DAG"`
does not work because because kotlin expects an expression after `tema = `, and assignments are not considered expressions.

`tema = Tema().setValue("DAG")` wont work because the setValue method has return-type void/Unit. 

`tema = Tema("DAG")` wont work because these generated objects don't have constructors that take arguments.

Reply:
😢